<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WIFI_frame</class>
 <widget class="QWidget" name="WIFI_frame">
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="SerialBox5_lite" name="serial_widget" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="caption_label"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="3">
         <widget class="QLineEdit" name="le_Password"/>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="le_Gate"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>password</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="le_Network"/>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="le_RemoteIP"/>
        </item>
        <item row="3" column="3">
         <widget class="QComboBox" name="cb_EncryptType"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>gate</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>remote port</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>encrypt type</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>mask</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>IP</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="le_IP"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>network</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>remote IP</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="le_RemotePort"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="le_Mask"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_client">
        <property name="text">
         <string>create client</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_server">
        <property name="text">
         <string>create server</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_send_data">
        <property name="text">
         <string>send data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_read_settings">
        <property name="text">
         <string>read settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LogBox" name="logBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogBox</class>
   <extends>QTextEdit</extends>
   <header>logbox.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SerialBox5_lite</class>
   <extends>QWidget</extends>
   <header>serialbox5_lite.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_Network</tabstop>
  <tabstop>le_Password</tabstop>
  <tabstop>le_IP</tabstop>
  <tabstop>le_Gate</tabstop>
  <tabstop>le_Mask</tabstop>
  <tabstop>le_RemoteIP</tabstop>
  <tabstop>le_RemotePort</tabstop>
  <tabstop>cb_EncryptType</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
